templates/plugin-gruntfile.mustache

```diff
@@ -11,10 +11,11 @@ module.exports = function( grunt ) {
 			options: {
 				textdomain: '{{textdomain}}',
 			},
-			target: {
-				files: {
-					src: [ '*.php', '**/*.php', '!node_modules/**', '!php-tests/**', '!bin/**' ]
-				}
+			update_all_domains: {
```

Comment:
> Not sure if the recommended should be forcing all textdomains to be the same by default. Maybe this change is not really necessary?

It seems like a reasonable assumption to me that all textdomains are the same for a given project. I'll wait for the go-ahead from you though.